sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/query.sql

```diff
@@ -0,0 +1,57 @@
+-- Aggregated clients data including active users, new profiles and search metrics
+SELECT
+  activity_segment AS segment,
+  app_version AS app_version,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  attribution_medium,
+  attribution_source,
+  attribution_variation,
+  city,
+  country,
+  device_model,
+  distribution_id,
+  is_default_browser,
+  locale,
+  normalized_app_name AS app_name,
+  normalized_channel AS channel,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
+  COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
+  COUNT(DISTINCT client_id) AS mau,
+  COUNT(DISTINCT IF(is_new_profile IS TRUE, client_id, NULL)) AS new_profiles,
```

Comment:
Yes, we didn't considered in the initial scope for this aggregation. Since I am deploying the schema and backfilling, I added uri_count and active_hours already. As for dev tools would need to bring it to unified metrics first.